Assets/SEE/XR/RadialSelection.cs

```diff
@@ -0,0 +1,409 @@
+            }
+            if (actions[i] == "HideAll")
+            {
+                HideMode = HideModeSelector.HideAll;
+                GlobalActionHistory.Execute((ActionStateType)ActionStateTypes.AllRootTypes.AllElements().FirstOrDefault(a => a.Name == "Hide"));
+                return;
+            }
+            if (actions[i] == "HideSelected")
+            {
+                HideMode = HideModeSelector.HideSelected;
+                GlobalActionHistory.Execute((ActionStateType)ActionStateTypes.AllRootTypes.AllElements().FirstOrDefault(a => a.Name == "Hide"));
+            }
+            if (actions[i] == "HideUnselected")
+            {
+                HideMode = HideModeSelector.HideUnselected;
+                GlobalActionHistory.Execute((ActionStateType)ActionStateTypes.AllRootTypes.AllElements().FirstOrDefault(a => a.Name == "Hide"));
+            }
+            if (actions[i] == "HideOutgoing")
+            {
+                HideMode = HideModeSelector.HideOutgoing;
+                GlobalActionHistory.Execute((ActionStateType)ActionStateTypes.AllRootTypes.AllElements().FirstOrDefault(a => a.Name == "Hide"));
+            }
+            if (actions[i] == "HideIncoming")
+            {
+                HideMode = HideModeSelector.HideIncoming;
+                GlobalActionHistory.Execute((ActionStateType)ActionStateTypes.AllRootTypes.AllElements().FirstOrDefault(a => a.Name == "Hide"));
+            }
+            if (actions[i] == "HideAllEdgesOfSelected")
+            {
+                HideMode = HideModeSelector.HideAllEdgesOfSelected;
+                GlobalActionHistory.Execute((ActionStateType)ActionStateTypes.AllRootTypes.AllElements().FirstOrDefault(a => a.Name == "Hide"));
+            }
+            if (actions[i] == "HideForwardTransitiveClosure")
+            {
+                HideMode = HideModeSelector.HideForwardTransitiveClosure;
+                GlobalActionHistory.Execute((ActionStateType)ActionStateTypes.AllRootTypes.AllElements().FirstOrDefault(a => a.Name == "Hide"));
+            }
+            if (actions[i] == "HideBackwardTransitiveClosure")
+            {
+                HideMode = HideModeSelector.HideBackwardTransitiveClosure;
+                GlobalActionHistory.Execute((ActionStateType)ActionStateTypes.AllRootTypes.AllElements().FirstOrDefault(a => a.Name == "Hide"));
+            }
+            if (actions[i] == "HideAllTransitiveClosure")
+            {
+                HideMode = HideModeSelector.HideAllTransitiveClosure;
+                GlobalActionHistory.Execute((ActionStateType)ActionStateTypes.AllRootTypes.AllElements().FirstOrDefault(a => a.Name == "Hide"));
+            }
+            if (actions[i] == "HighlightEdges")
+            {
+                HideMode = HideModeSelector.HighlightEdges;
+                GlobalActionHistory.Execute((ActionStateType)ActionStateTypes.AllRootTypes.AllElements().FirstOrDefault(a => a.Name == "Hide"));
+            }
+            else
```

Comment:
I may be wrong, but I think there could be a bug here. This `else` statement is always executed when `actions[i]` is not `HighlightEdges`. That is, this else statement is also triggered when, for example, the `HideAll` if block has run. Is this intended, or should these if statements all be `else if`s?